assets/js/modules/analytics/components/common/PropertySelect.js

```diff
@@ -64,7 +71,7 @@ export default function PropertySelect() {
 		}
 	}, [ propertyID ] );
 
-	if ( ! hasResolvedAccounts || ( accountID && ! hasResolvedProperties ) ) {
+	if ( ! hasResolvedAccounts || ( hasStartedFetchingProperties && ! hasResolvedProperties ) ) {
```

Comment:
Why is this extra check for whether resolving has already started necessary? If there's nothing selected, shouldn't the resolver still have to be resolved either way (it would be the resolver for e.g. a value of `undefined`).

Reply:
I have updated it to use `isResolving` which is more accurate and strictly identifies when we are fetching something or not. It helps us to show the progress bar only when an actual process of fetching is happening.